crowdsourcer/management/commands/import_foi_data.py

```diff
@@ -0,0 +1,323 @@
+            "section": "Governance & Finance",
+            "question": 9,
+            "type": "yes_no",
+        },
+        "Planning 11": {
+            "section": "Planning & Land Use",
+            "question": 11,
+            "type": "yes_no",
+        },
+        "Transport 11ii": {
+            "section": "Transport",
+            "question": 11,
+            "type": "yes_no",
+        },
+    }
+
+    tiered_options = {
+        "B&H Q2": {
+            "options": [
+                {"description": "100% green energy", "score": 1},
+                {"description": "Green Tariff", "score": 1},
+                {"description": "Creates 20% own energy", "score": 1},
+                {"description": "Criteria not met", "score": 0},
+                {"description": "No response", "score": 0},
+            ]
+        },
+        "B&H Q3": {
+            "options": [
+                {"description": "50% or above", "score": 1},
+                {"description": "60% or above", "score": 2},
+                {"description": "90% or above", "score": 3},
+                {"description": "Criteria not met", "score": 0},
+                {"description": "No response", "score": 0},
+            ]
+        },
+    }
+
+    authority_name_map = {
+        "Lincoln City Council": "City of Lincoln",
+        "King's Lynn and West Norfolk Borough Council": "Kings Lynn and West Norfolk Borough Council",
+        "Common Council of the City of London": "City of London",
+        "London Borough of Barking and Dagenham Council": "Barking and Dagenham Borough Council",
+    }
+
+    def get_council_lookup(self):
+        url = get_dataset_url(
+            repo_name="uk_local_authority_names_and_codes",
+            package_name="uk_la_future",
+            version_name="1",
+            file_name="uk_local_authorities_future.csv",
+            done_survey=True,
+        )
+        df = pd.read_csv(url)
+
+        lookup = {}
+        for _, row in df.iterrows():
+            gss = row["gss-code"]
+            lookup[row["nice-name"].replace(" ", "-").lower()] = gss
+            lookup[row["nice-name"]] = gss
+            lookup[row["official-name"]] = gss
+
+        return lookup
+
+    def get_option_for_question(self, q, option):
+        option = Option.objects.get(question=q, description=option)
+
+        return option
+
+    def get_bh_3_answer(self, q, row, value):
+        if value == 0:
+            description = "Criteria not met"
+        elif value == 1:
+            try:
+                percentage = row.iloc(8)
+            except ValueError:
+                percentage = 50
+
+            threshold = 50
+            if percentage >= 90:
+                threshold = 90
+            elif percentage >= 60:
+                threshold = 60
+
+            description = f"{threshold}% or above"
+
+            option = Option.objects.get(question=q, description=description)
+
+            return option
+
+    def get_bh_2_answer(self, q, row, value):
```

Comment:
My reading, if the normal value is 1, green energy is used, but then the extra details column has information about how much green energy the council makes itself?